src/contexts/ThemeContext/ThemeContextProvide.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import { PropsWithChildren, useEffect, useState } from 'react';
+import { ThemeContext } from './ThemeContext';
+
+export function ThemeProvider({ children }: PropsWithChildren) {
+  const [theme, setTheme] = useState('light');
+
+  const toggleTheme = (theme: string) => {
```

Comment:
It doesn’t look like a toggle, more like a set function. So you can change the logic to use it as a real toggle or remove it and pass setTheme to the context